manifests/sar_metric.pp

```diff
@@ -0,0 +1,59 @@
+define puppet_metrics_collector::sar_metric (
+  String                    $output_dir,
+  String                    $scripts_dir,
+  Enum['absent', 'present'] $metric_ensure  = 'present',
+  String                    $metrics_type   = $title,
+  String                    $cron_minute    = '*/5',
+  Integer                   $retention_days = 90,
+  Integer                   $polling_frequency_seconds = 1,
+  Integer                   $collection_frequency = 5, #minutes
```

Comment:
since the polling frequency parameter indicates its value is in seconds, maybe the collection frequency should indicate it is in minutes?

Reply:
This was to maintain compatibility with the init.pp... That doesn't have a polling freq, so I was free to add units to the name.  collection was already there.  If I change it here, when this gets merged into init, it won't match...